packages/scripts/scripts/check-licenses.js

```diff
@@ -65,6 +65,7 @@ const gpl2CompatibleLicenses = [
 	'(MIT AND BSD-3-Clause)',
 	'(MIT AND Zlib)',
 	'(CC-BY-4.0 AND MIT)',
+	'0BSD',
```

Comment:
@ntwb or @pento – can you perform sanity check for this new license type?

Reply:
[0BSD](https://spdx.org/licenses/0BSD.html) is [ISC](https://spdx.org/licenses/ISC.html) without the requirement to maintain the copyright notice. It's about as close to public domain as you can get. While the FSF haven't commented on the compatibility of 0BSD, they have stated that [ISC is GPL-compatible](https://www.gnu.org/licenses/license-list.html#ISC), which is enough for me.